test/AutoRest.TestServer.Tests/Mgmt/TestProjects/TestProjectTests.cs

```diff
@@ -42,6 +44,51 @@ public void ValidateMethodExists(string methodName)
             }
         }
 
+        [Test]
+        public void ValidateResourceGroupExtensions()
+        {
+            if (_projectName.Equals(""))
+            {
+                return;
+            }
+            Type[] allTypes = Assembly.GetExecutingAssembly().GetTypes();
+
+            Type resourceExtensions = allTypes.FirstOrDefault(t => t.Name == "ResourceGroupExtensions" && t.Namespace == _projectName);
+            Assert.NotNull(resourceExtensions);
+
+            // TODO: Bring below tests back after container deploy
```

Comment:
uncomment this and add [Ignore("TODO ADO item 1111")] above the test method

This will remind us that there is a "skipped" test vs seeing the test as green when its actually not doing anything

Reply:
Done